tmt/steps/__init__.py

```diff
@@ -68,8 +68,25 @@
 
 
 DEFAULT_ALLOWED_HOW_PATTERN: Pattern[str] = re.compile(r'.*')
-DEFAULT_PLUGIN_METHOD_ORDER: int = 50
 
+#
+# Following are default and predefined order values for various special phases
+# recognized by tmt. When adding new special phase, add its order below, and
+# do not forget to update the list in /spec/core/order. Please, keep the name
+# prefix, so it's easy to find all `PHASE_ORDER_*` constants for documentation.
+#
+
+#: The default order of any object.
+# TODO: this is a duplication of tmt.base.DEFAULT_ORDER. Unfortunately, tmt.base
+# imports tmt.steps, not the other way around.
+# `PHASE_ORDER_DEFAULT = tmt.base.DEFAULT_ORDER` would be way better.
+PHASE_ORDER_DEFAULT = 50
+#: Installation of essential plugin and check requirements.
+PHASE_ORDER_INSTALL_ESSENTIAL_REQUIRES = 30
```

Comment:
```suggestion
PREPARE_PHASE_ORDER_INSTALL_ESSENTIAL_REQUIRES = 30
```

These are not generic for all phases, should we be more specific?

Reply:
Thanks!